mypy/build.py

```diff
@@ -859,8 +859,12 @@ def find_modules_recursive(module: str, lib_path: List[str]) -> List[BuildSource
 
 def verify_module(id: str, path: str) -> bool:
     """Check that all packages containing id have a __init__ file."""
+    if path.endswith(tuple(PYTHON_EXTENSIONS)):
```

Comment:
I think you should probably rewrite the rest of this function, and update the docstring, as this change makes the docstring incorrect, and much of the rest of the function duplicative (eg `path.endswith(('__init__.py', '__init__.pyi'))` will never be reached...